### PR TITLE
Colour Scheming: Stats - Turn Level 3 Text Colour White

### LINKED ISSUE #30216: Accessibility: Low contrast in All Time Views table
Some of the values in the All Time Views stats table are very low contrast, with dark grey text on a fairly dark blue background, e.g.

image

#user-report

1714206-zen

### PR DESCRIPTION
**Pull request by torres126**
#### Changes proposed in this Pull Request

Let the text colour for level 3 stats (dark grey) be white in favour of it being easier to read. 

#### Testing instructions

Does the change look as expected? 

**Before:**

dfhgfhgdfdgh

**After:**

gsdfsfgdsdfg

(cc @flootr, @blowery, @drw158) 

Fixes #30216

### CODE REVIEW

File: client/my-sites/stats/stats-views/style.scss [hunk 1]
```diff
@@ -49,6 +49,7 @@
 
 	&.level-3 {
 		background-color: var( --color-primary-light );
+		color: white;
```
Comment (flootr) [inline client/my-sites/stats/stats-views/style.scss]: let's use `var( --color-white )` to make sure we can switch it for different color themes

Comment (torres126) [reply client/my-sites/stats/stats-views/style.scss]: Should the same be done [here](https://github.com/Automattic/wp-calypso/blob/5384b0e48a082ce46f249a1510982fc054cb47b0/client/my-sites/stats/stats-views/style.scss#L57) and [here](https://github.com/Automattic/wp-calypso/blob/5384b0e48a082ce46f249a1510982fc054cb47b0/client/my-sites/stats/stats-views/style.scss#L62) as well?

Comment (flootr) [reply client/my-sites/stats/stats-views/style.scss]: Good point, yes! 👍

Comment (torres126) [reply client/my-sites/stats/stats-views/style.scss]: Addressed! :+1:

Comment (blowery) [reply client/my-sites/stats/stats-views/style.scss]: @torres126 yup! thanks!

**Pull request merged**